README.md

```diff
@@ -71,8 +140,6 @@ In an example above, ansible was instructed to use specific private key (overrid
 
 Other playbooks from playbooks/ directory can be used similarly, though some might need more variables and investigating their parameters is recommended first.
 
-More thorough guides can be found in the [docs folder](https://github.com/theforeman/forklift/tree/master/docs).
```

Comment:
Perhaps we should change this to https://theforeman.github.io/forklift/ instead and give it a better landing page.

Reply:
Changed docs/index.md to link to README.md -- how does that work for you?

Reply:
Still not a good idea, because we should encourage people to see all the documentation. In my mind that means less README. Basically none IMHO. Because if there are some instructions, people may think it's all the documentation and not look further.

I'd expect:
* A quickstart: like 5 lines of code to get started
* Required versions
* Where to find the full documentation.

Reply:
The full documentation contains a lot of information that is only going to be relevant to a smaller cross section of users, so compelling people to refer to the full documentation in the beginning poses an additional burden of sifting through all that information and determining what is relevant. This also means sifting through multiple documents, which shouldn't be required for getting started with basic usage.

On the other hand, the quickstart can provide both 1. enough information that it's realistically possible to get started using the tool, and 2. clear and prominent indication that complete documentation exists, and where to find it once it becomes necessary.

This approach is in line with the best practices outlined [here](https://swimm.io/blog/get-started-with-technical-documentation-writing-best-practices/), in particular numbers 2, 3, and 5.

I think there is information that can certainly be pruned from the README, but I'm hesitant to remove from the sections on the basic usage of the development and production boxes, which are the most important IMO. It's not super useful to know how to `vagrant up` if you don't actually know how to start the development server or access the WebUI afterwards.

What do you think?

Reply:
I don't mind having one long document, but rather than symlinking I'd suggest `mv README.md docs/index.md` and then write a minimal new README that redirects to the docs site.

Reply:
Is your primary concern to make it so that the docs site is highly discoverable to those looking at this repo in their browser?

If so, I agree that's important. Can it be adequately addressed by making the link to the docs site be the most prominent item at the very beginning of README.md, without stripping the rest of its content?

For a terminal user, locating the quickstart guide only in docs/index.md means that it's less obvious to start here, vs. the top level README.md, while a README.md which contains only redirects to the docs site is not as useful in the terminal.

From my PoV then, adding the link the docs site prominently at the top of README.md and creating docs/index.md as a symlink then combines the best features of both approaches.

That said, I don't really need to die on this hill, and I'd prefer to try merging this sooner rather than later (as I said, we're onboarding a new developer at present, and another in the near term) so if I've failed to convince you just let me know and I'll make the change.